templates/base.html.twig

```diff
@@ -11,23 +11,25 @@
 	<meta name="description" content="{{ instelling('stek', 'beschrijving') }}">
 	<meta name="google-site-verification" content="zLTm1NVzZPHx7jiGHBpe4HeH1goQAlJej2Rdc0_qKzE"/>
 	<meta name="apple-itunes-app" content="app-id=1112148892, app-argument={{ app.request.uri }}">
+	<meta property="og:type" content="article">
 	<meta property="og:url" content="{{ app.request.uri }}"/>
 	<meta property="og:title" content="C.S.R. Delft | {{ block('titel') }}"/>
 	<meta property="og:locale" content="nl_nl"/>
-	<meta property="og:image" content="{{ absolute_url(asset_url('images/beeldmerk.png')) }}"/>
+	<meta property="og:image" content="{{ absolute_url('/images/beeldmerk.png') }}"/>
```

Comment:
De afbeeldingen die door webpack worden gehaald komen in `/dist/images/<randomstring>.png` gestopt. `asset_url` zoekt de goede url op bij het bestand, de bestandsnaam kan veranderen als de inhoud van het bestand veranderd zodat we ons niet druk hoeven te maken over cachen.

Met deze verandering verwijst ie naar `https://csrdelft.nl/images/beeldmerk.png` en dit is een 404.

Reply:
Er zijn bestanden die niet door webpack gehaald worden en deze staan in `/images` en kunnen wel met alleen `absolute_url` gebruikt worden, kan een beetje verwarrend zijn.

Idealiter worden alle bestanden door webpack gehaald, want dan hoef je je niet zo druk te maken over de kwaliteit van bestanden en kun je makkelijk schakelen van compressiealgoritme.